File: C/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
   int n = 0;
    while (n < 1 || n > 8) 
    {
        n = get_int("Height: ");
    }
    
   for (int i = 0; i < n; i++)
   {
      for (int j = 0; j < n; j++)
      {
          if (j < n - i - 1)
          {
             printf(" ");
          }
          else 
        { 
           printf("#");
        }
        
      }
       printf("\n");
   }
}
// j = n - (i + 1)

File: Python/mario.py
from cs50 import get_string
from cs50 import get_int

height = 0
while height < 1 or height > 8:
    
    height = get_int("Height: ")

for i in range(height):

    for j in range(height):

        if j < height - 1 - i:

            print(" ", end="")

        else:

            print("#", end="")

    print()

